routes/messages.routes.js

```diff
@@ -9,13 +10,28 @@ router.get("/hello-world", (req, res) => {
 
 router.post("/messages", (req, res) => {
 
-    const {destination, message} = req.body
-    messagesService
-        .sendMessage({destination, message})
-        .then(({data}) => {
-            res.status(200).json(data)
-        })
-        .catch((err) => res.status(500).json({ message: "The service could not store the message", errorData: err }))
-})
+    const { destination, message } = req.body
+
+    if (!req.body.destination) {
+        res.status(400).json({ message: 'Destination field is required' })
+    } else if (!req.body.message) {
+        res.status(400).json({ message: 'Message field is required' })
```

Comment:
Nice I think 400 is good here, do you think thatt 422 status code would be better?

Reply:
Thank you very much it is corrected now